saleor/order/migrations/tasks/saleor3_13.py

```diff
@@ -46,3 +51,63 @@ def order_events_rename_transaction_capture_events_task():
     if ids:
         update_type_to_transaction_charge_requested(qs)
         order_events_rename_transaction_capture_events_task.delay()
+
+
+@app.task
+def update_orders_charge_statuses_task():
+    orders = Order.objects.filter(
+        Exists(OrderGrantedRefund.objects.filter(order_id=OuterRef("pk")))
+    )
+    orders = orders.annotate(
+        refunded_amount=Coalesce(
+            Sum("granted_refunds__amount_value"),
```

Comment:
Did you check maybe if doing `Subquery` and annotate there have better performance?